catalog/ui/search-ui/search-redirect/pom.xml

```diff
@@ -27,12 +27,6 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.core</artifactId>
         </dependency>
-        <dependency>
```

Comment:
❓ If we are going to import packages from jetty-servlets bundle, wouldn't we still want jetty-servlets as a dependency?

Reply:
in the original PR (https://github.com/codice/ddf/pull/2159/files), it seems as though the dependency was only added to help with GzipFilter functionality. After these changes, it was still working. Therefore, it seems unnecessary at this point.